InternshipApp/InternshipApp/FlightsApp/View/Screens/ChooseSeatScreen.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ChooseSeatScreen.swift
+//  InternshipApp
+//
+//  Created by Sead Mašetić on 3. 9. 2024..
+//
+
+import SwiftUI
+
+struct ChooseSeatScreen: View {
+    @ObservedObject var flightViewModel: FlightViewModel
+
+    var body: some View {
+        NavigationStack {
+            VStack {
+                Text("Select Your Seat")
+                    .font(.largeTitle)
+                    .fontWeight(.bold)
+                    .padding(.top)
+
+                ScrollView {
+                    VStack(spacing: 16) {
+                        ForEach(flightViewModel.seatNumbers, id: \.self) { seat in
```

Comment:
again - don't use ".self" as an identifier here. 
When you do, it means that the unique identifier for each row will be a view itself which is not so "unique" :)